mxcube3/routes/Login.py

```diff
@@ -172,6 +172,12 @@ def get_initial_state():
     except AttributeError:
         data['current_phase'] =  'None' # in case the diff does not have this implemented
 
+    try:
```

Comment:
This block of code should be in a beamline/site specific file, I think its in the beamline setup hardware 
object today. Perhaps you already thought about it and its just a temporary fix ?